app/models/user.rb

```diff
@@ -3,8 +3,12 @@ class User < ApplicationRecord
          :recoverable, :rememberable, :trackable, :validatable
   validates :name, :surname, presence: true,
                              length: { in: 3..30 },
-                             format: { with: /\A[a-zA-Z]+\z/ }
+                             format: { with: /\A[a-zA-Z]+\z/ },
+                             unless: :skip_register_validation
   validates :phone_number, presence: true,
                            numericality: true,
-                           length: { is: 13 }
+                           length: { is: 13 },
+                           unless: :skip_register_validation
```

Comment:
same here

Reply:
the same here
![screenshot from 2018-04-10 16-32-45](https://user-images.githubusercontent.com/15639895/38610174-b8a6d99e-3d88-11e8-8d53-69383ea855cf.png)